### PR TITLE
Rename and moved directory

### LINKED ISSUE #71: Move `client` directory into `packages` and rename `client` to `website`
### 🔍 Overview

I notice most monorepos that's using Yarn Workspace have all the projects into `packages` directory. So thought we would follow that convention where I think that keeps things clean while on the root directory we have all the configurations for CI, lints, Husky, etc. And for renaming the `client` to `website` would make more sense. Since we "might" want a React Native project in the future?

### 📄 Steps

1. Move the `client` directory into `packages`
2. Rename `client` directory into `website`
3. Rename the package name from `package.json` inside `website` directory to `"website"` (Which previously was `"client"`)
4. On the root directory, change all references from `"client"` workspace to `"website"` (e.g. `"yarn workspace website dev"`)

### ⚠️  Note

Only take action on this ticket when most of the outstanding pull requests have been merged in, that way we can avoid unnecessary merge conflicts.
- #70 
- #64 
- #63 
- #41

The other pull requests were inactive so I'll mention them here separately:
- #38 
- #50

### PR DESCRIPTION
**Pull request by carlomigueldy**
### Overview

This changes the root directory of the project thus it would require to update Vercel's build configuration and root directory path as well.

Closes #71

### CODE REVIEW

**Comment by carlomigueldy** [comment]
I think the build is failing on Vercel since the build path needs to get updated while I updated the directory and workspace name.

File: packages/website/styles/ui-components/Chakra-Button.tsx [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      fontSize="14px"
+      onClick={onClick}
+      as={as}
+    >
+      {children}
+    </Button>
+  );
+};
+
+export const ButtonGray2 = ({ children, as, onClick }: ChakraButtonProps) => {
```
Comment (frankTurtle) [inline packages/website/styles/ui-components/Chakra-Button.tsx:44]: idk if this is new or moved, but `ButtonGray2` should be renamed to what's different about it vs `ButtonGray`

File: packages/website/styles/ui-components/Chakra-Button.tsx [hunk 2]
```diff
@@ -0,0 +1,156 @@
+import { Button, ButtonGroup } from '@chakra-ui/react';
```
Comment (frankTurtle) [inline packages/website/styles/ui-components/Chakra-Button.tsx:1]: A lot of duplicate code, imo

**Review by frankTurtle — APPROVED** [review]
Nothing too crazy, PR doesn't appear to break anything.

Some refactoring might be useful ( see some comments ), but it's nothing crucial.

**Comment by carlomigueldy** [comment]
> Nothing too crazy, PR doesn't appear to break anything.
> 
> Some refactoring might be useful ( see some comments ), but it's nothing crucial.

Thanks for reviewing the PR despite having multiple files changed detected 😆  @frankTurtle 

Yeah some refactoring could be useful though I'll prefer it to tackle in a separate PR since most of those duplicated code are not made in this PR, I'll create an issue for it

**Comment by carlomigueldy** [comment]
This PR is dependent on most PRs, I'll have to might re-do everything I guess or maybe close this

**Comment by carlomigueldy** [comment]
Closing this and will re-open a branch since my current branched is messed up.

**Pull request closed without merging**